applications/serial_lte_modem/sample.yaml

```diff
@@ -46,17 +50,22 @@ tests:
     integration_platforms:
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
-    tags: ci_build sysbuild
+    tags:
+      - ci_build
+      - sysbuild
   applications.serial_lte_modem.lwm2m_carrier.softbank:
     sysbuild: true
     build_only: true
-    extra_args: EXTRA_CONF_FILE="overlay-carrier.conf;overlay-carrier-softbank.conf"
-      SB_EXTRA_CONF_FILE=sysbuild-softbank.conf
+    extra_args:
+      - EXTRA_CONF_FILE="overlay-carrier.conf;overlay-carrier-softbank.conf"
+      - SB_EXTRA_CONF_FILE=sysbuild-softbank.conf
     platform_allow:
       - nrf9160dk/nrf9160/ns
     integration_platforms:
       - nrf9160dk/nrf9160/ns
-    tags: ci_build sysbuild
+    tags:
+      - ci_build
+      - sysbuild
   applications.serial_lte_modem.lwm2m_carrier.lgu:
```

Comment:
The `extra_args` list under `applications.serial_lte_modem.lwm2m_carrier.lgu` is still space-separated